sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -606,6 +642,39 @@ public void close() {
         return getEntityWithResponse(partitionKey, rowKey, select).flatMap(FluxUtil::toMono);
     }
 
+    /**
```

Comment:
nit: aggregating methods by visibility. I see an interspersing of public, package-private, and private methods. When I browse code, usually I hit up the public methods first (at the top) and then go down.

Reply:
Gotcha, in this case the client is organized by operation - each operation has a bunch of public overloads, then a few more that also provide the response, and all of them eventually just call the package-private method that actually performs the operation. I can look at re-organizing a bit in a future PR.